src/buildHTML.js

```diff
@@ -1232,6 +1227,18 @@ groupTypes.delimsizing = function(group, options) {
             [group.value.mclass]);
 };
 
+groupTypes.verb = function(group, options) {
+    var text = buildCommon.makeVerb(group, options);
+    var body = [];
+    var i;
+    for (i = 0; i < text.length; i++) {
+        body.push(buildCommon.makeSymbol(
+            text[i], "Main-Regular", group.mode, options, ["mathtt"]));
+    }
+    buildCommon.tryCombineChars(body);
+    return makeSpan(["mord", "verb"], body, options);
```

Comment:
Both good points.  I was imagining a user might want to style `\verb`s in some special way (like Github here does with `backquotes`), but it's a pretty hypothetical situation, and maybe not appropriate for LaTeX formatting, so probably not worth keeping this class.  I'll remove it.